apps/cancel-request/fields/index.js

```diff
@@ -8,29 +8,31 @@ module.exports = {
   },
   'cnc-main-applicant-full-name': {
     mixin: 'input-text',
-    validate: ['required', { type: 'maxlength', arguments: 250 }, { type: 'minlength', arguments: 3 }],
+    validate: ['required', { type: 'maxlength', arguments: 250 }, { type: 'minlength', arguments: 3 }, 'notUrl'],
     legend: {
-      className: 'bold'
+      className: ['govuk-input', 'govuk-!-width-two-thirds']
     }
   },
   'cnc-main-applicant-dob': dateComponent('cnc-main-applicant-dob', {
     mixin: 'input-date',
     legend: {
       className: 'bold'
     },
-    validate: ['required']
+    validate: ['required', 'date',
+      { type: 'after', arguments: ['120', 'years'] },
+      { type: 'before', arguments: ['0', 'days']}]
   }),
   'cnc-main-applicant-nationality': {
     mixin: 'select',
-    validate: 'required',
+    validate: ['required', 'notUrl'],
```

Comment:
Don't think we need notUrl validation for applicant nationality as the user can't enter anything that's not in the country list